lambda/update/statutory_waiting_period.go

```diff
@@ -20,7 +20,7 @@ func (r StatutoryWaitingPeriod) Apply(lpa *shared.Lpa) []shared.FieldError {
 	}
 
 	for _, a := range lpa.Attorneys {
-		if a.Status != shared.AttorneyStatusInactive && (a.SignedAt == nil || a.SignedAt.IsZero()) {
+		if a.Status != shared.AttorneyStatusRemoved && (a.SignedAt == nil || a.SignedAt.IsZero()) {
```

Comment:
would the same need to be checked for trust corporations?

Reply:
Yep, added